server/common/HatoholArmPluginInterfaceHAPI2.cc

```diff
@@ -46,81 +46,39 @@ std::list<HAPI2ProcedureDef> defaultValidProcedureList = {
 	{PROCEDURE_HAP,    "fetchEvents",               HAP_OPTIONAL},
 };
 
-class HatoholArmPluginInterfaceHAPI2::AMQPHAPI2MessageHandler
-  : public AMQPMessageHandler
-{
-public:
-	AMQPHAPI2MessageHandler(HatoholArmPluginInterfaceHAPI2 &hapi2)
-	: m_hapi2(hapi2)
-	{
-	}
-
-	enum class JsonRpcObjectType {
+struct JsonRpcObject {
```

Comment:
In other code, we use capital letters for abbreviated words such as JSON.
Here, we should use JSON_RPCObject or JSON_RPC_Object ?
But I feel a little uncomfortable...
JsonRpcObject looks simpler than JSON_RPCObject...